src/js/gallery/service/resize_emulator.coffee

```diff
@@ -1,46 +1,33 @@
-angular.module('multiGallery').service 'ResizeEmulator', [ '$window', ($window)->
-
-  class ResizeEmulator
-
-    @_storage = []
-
-    constructor: ->
-      @_storage = []
-      angular.element($window).bind('resize orientationchange', => @onResize())
-
-    onResize: ->
-      if @_storage.length > 0
+angular.module('cycleGallery').factory 'ResizeEmulator', [
+  'ResizeEmulatorAction', '$window',
+  (ResizeEmulatorAction, $window)->
+
+    class ResizeEmulator
+
+      constructor: ->
+        @_storage = []
+        angular.element($window).bind('resize orientationchange', => @onResize())
+
+      onResize: ->
+        if @_storage.length > 0
+          for action in @_storage
+            action.apply()
+
+      bind: (fn, key) ->
+        action = @actionExists(key)
+        if action
+          action.fn = fn
+        else
+          action = new ResizeEmulatorAction(fn, key)
+          @_storage.push(action)
+
+      actionExists : (key)->
```

Comment:
Colon assignment without proper spacing<br>Function arrow (->) must be spaced properly